tests/letstest/targets.yaml

```diff
@@ -3,6 +3,17 @@
 targets:
   #-----------------------------------------------------------------------------
   #Ubuntu
+  - ami: ami-0758470213bdd23b1
+    name: ubuntu20.04
```

Comment:
Nice! This PR basically resolves https://github.com/certbot/certbot/issues/7915 too.

In this or another PR, I'd kind of like to add this amd64 image to `apache2_targets.yaml` but otherwise it's resolved in my book.